nsv13/code/modules/overmap/knpc.dm

```diff
@@ -436,6 +458,12 @@ This is to account for sec Ju-Jitsuing boarding commandos.
 	var/dist = get_dist(H, target)
 	//We're holding a gun. See if we can shoot it....
 	HA.pathfind_to(target) //Walk up close and YEET SLAM
+	var/default_view = getviewsize(CONFIG_GET(string/default_view))
+	var/viewdist = list(round(default_view[1]/2), round(default_view[2]/2))
+	var/is_in_x_view = ISINRANGE(target.x, H.x - viewdist[1], H.x + viewdist[1])
+	var/is_in_y_view = ISINRANGE(target.y, H.y - viewdist[2], H.y + viewdist[2])
+	if (!(is_in_x_view && is_in_y_view))
```

Comment:
Shallow binary implementation.

Reply:
Shallow binary? What's that

Reply:
Ah, I think Karmic was pointing out that you're using a NAND evaluation, which doesn't look right here.
NAND inverts the output of AND, not the inputs.
![image](https://user-images.githubusercontent.com/59686430/184277761-712facd0-ee2e-44a1-b9f6-057285747a32.png)
As you can see, if x and y are in view the condition will be true and the proc will return, which doesn't appear to be the desired behavior